ecosystem/indexer-grpc/indexer-transaction-generator/src/managed_node.rs

```diff
@@ -114,8 +117,14 @@ impl ManagedNode {
                 LocalnetNodeType::BuiltIn(join_set)
             },
         };
+
+        let transaction_stream_url = Url::parse("http://localhost:50051").unwrap();
```

Comment:
Better to use 127.0.0.1, some systems do weird things with localhost (like use the ipv6 address).

Reply:
Also maybe use the `LOCAL_INDEXER_GRPC_URL` variable you define later.